### ISSUE TITLE
tensorflow's read_data_sets would yield a validation set with the same labels

### ISSUE THREAD

**Issue by shaform** [zalandoresearch/fashion-mnist#3]
>from tensorflow.examples.tutorials.mnist import input_data
> data = input_data.read_data_sets('data/fashion', one_hot=True)

the validation set only contain 0 as labels

**Comment by kashif** [comment]
@shaform will check and report back.

**Comment by hanxiao** [comment]
@shaform Thanks for reporting. We didn't shuffle the data when storing it, resulting all 0 labels first, followed all 1s, all 2s, ... This is good for retrieving examples of certain labels, also giving a smaller `labels-idx1-ubyte.gz `. Tensorflow MNIST loader takes the first/last 5000 examples for the validation set. That's why you see examples have the same label.

**Comment by shaform** [comment]
@hanxiao Thank you. It's indeed the case. Although it's a bit inconvenient, I think users might still be able to split the data by themselves. But I do wish there is official split for validation, just as many people would use the default split provided by tensorflow.
